app/controllers/group_members_controller.rb

```diff
@@ -0,0 +1,72 @@
+class GroupMembersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    group = Group.find(params[:group_id])
+
+    res = current_user &&
+      (group.is_admin?(current_user) || group.is_mod?(current_user))
```

Comment:
`current_user` is used here as the `current_member` method is based on a GroupMember ID being passed as a param.